lambdas/helpers/dynamoDBHelper.js

```diff
@@ -56,15 +56,23 @@ const stripSiteIdFromHost = (host, appDomain) => {
   throw new Error(`Unable to strip siteId from @host: ${host}`);
 };
 
+const getQueryParams = (tableName, tableKey, tableKeyValue) => ({
+  TableName: tableName,
+  Key: {
+    [tableKey]: tableKeyValue,
+  },
+});
+
 const getSiteQueryParams = (host, functionName) => {
-  const { tableName, siteKey, domain } = getAppConfig(functionName);
+  const { tableName, tableKey, domain } = getAppConfig(functionName);
   const siteKeyValue = stripSiteIdFromHost(host, domain);
-  return {
-    TableName: tableName,
-    Key: {
-      [siteKey]: siteKeyValue,
-    },
-  };
+  return getQueryParams(tableName, tableKey, siteKeyValue);
+};
+
+const getBuildQueryParams = (host, path, functionName) => {
```

Comment:
Does this mean there are 2 rows in the table for each site?

Reply:
there is row for each site + a row for each branch
= number of branches built + 1

Reply:
👍 so this is an ever-growing list?